docs/versionPolicy.md

```diff
@@ -0,0 +1,145 @@
+# Versioning
+
+Running `kustomize` means one is running a
```

Comment:
The program version and kustomization file version is related. The descriptions here and structure below make me feel that they are separate. 

Move some later session like 
```
 Kustomization Field Change Policy
```
outside of `Kustomization type versioning` may help better understanding.

Reply:
you're right!  tried to fix.